wollok-tests/src/wollok/imports/packageB/file2.wlk

```diff
@@ -0,0 +1,5 @@
+import packageA.file1.x
+
+object y{
+
+}
```

Comment:
I don't know if relative imports are working, maybe I should try them, I have not intended to do that.